4-forms-validations/AnnieFormEx3.html

```diff
@@ -0,0 +1,112 @@
+<p>Welcome to PintaYa Form
+    <br>
+    <img heiht = " 100" width = "100" src ="C:\Users\Bivi Anne\Documents\push-dev\html-tutorial\4-forms-validations/pinta.jpg  " alt = "PintaYa"/>
```

Comment:
You don't need spaces in between attributes and values. For the src, you can do:
```suggestion
    <img height="100" width="100" src="pinta.jpg" alt="PintaYa"/>
```